scss/_breadcrumb.scss

```diff
@@ -44,6 +45,44 @@
     }
   }
 
+  // Boosted mod: Handle responsive
+  display: none;
+
+  // Handle links differently to avoid any issue with colored backgrounds
+  a {
+    color: var(--#{$prefix}breadcrumb-color);
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  @include media-breakpoint-up(md) {
+    display: block;
+  }
+
+  @include media-breakpoint-down(md) {
+    &:first-child:last-child,
+    &:nth-last-child(2) {
+      display: block;
+    }
+
+    &:nth-last-child(2) {
+      padding-left: 0;
+
+      &::before {
+        float: left;
+        padding-right: 0 #{"/* rtl:ignore */"};
+        padding-left: var(--#{$prefix}breadcrumb-item-padding-x) #{"/* rtl:ignore */"};
+        color: var(--#{$prefix}breadcrumb-divider-color);
+        content: var(--#{$prefix}breadcrumb-divider, escape-svg($breadcrumb-divider)) #{"/* rtl:"} var(--#{$prefix}breadcrumb-divider, escape-svg($breadcrumb-divider-flipped)) #{"*/"};
+        transform: scaleX(-1) #{"/* rtl:"} scaleX(1) #{"*/"};
```

Comment:
Instead of transforming it automatically, I'm wondering if we shouldn't provide a `$breadcrumb-divider-mobile` (not sure about the naming). Because if you choose "/" as a divider, in mobile mode you probably don't want it to be "\".